index.bs

```diff
@@ -3169,7 +3181,7 @@ Updating Mature Publications on the Recommendation Track</h4>
 	</ul>
 
 	There is usually a formal review meeting
-	to ensure the requirements have been met before [=Director=]'s approval is given.
+	to ensure the requirements have been met before [=Team=]'s approval is given.
```

Comment:
I would prefer we did not use the phrase 'team approval'. I'm struggling with the phrasing "before the team verifies that the process requirements are met" is awfully wordy…